src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1942,5 +1942,12 @@ private void disposeDebugging(DisposeTypes type)
             }
         }
 
+        private void NotifyBackgroundChanged()
+        {
+            // NOTE
+            // Notify background modifications caused by one of BackgroundColor, BackgroundImage, Background and ClearBackground()
+            // By using reserved keyword "_background", user may get notified all background modifications.
+            NotifyPropertyChanged("_background");
```

Comment:
Just my curiosity :
Normally, when we add the string here for `PropertyChanged`, we use the Pascal case. like "Background".
Is there any reason for using snake_case like this with an underscore? 
Probably, is it to have different usage from general 'Background'?

Reply:
`View` has `Background` property (`PropertyMap` type) and it's using pascal case `Background` keyword! 
To distinguish from it I decided to add prefix "_" which can have private meaning. (it looks like snake case but it's just prefix. For example, `_somePrivateFieldName`)

Reply:
Ah~~ Okay, now I understood.
Thanks for the explanation :)